0008-security-recommendations/README.md

```diff
@@ -0,0 +1,98 @@
+---
+name: KP Security Recommendations
+shortname: 8/Security
```

Comment:
Non-descriptive shortname, looks like this is the ultimate security doc for everything. Make it "KP Security" or similar.